index.html

```diff
@@ -442,5 +443,25 @@ <h2>
       src="https://kit.fontawesome.com/e966a3bd22.js"
       crossorigin="anonymous"
     ></script>
+    <script>
```

Comment:
- [ ] Kindly place the JavaScript logic in a new file separated from the HTML document.